common/src/main/java/com/wynntils/features/user/IngredientPouchHotkeyFeature.java

```diff
@@ -2,31 +2,30 @@
  * Copyright © Wynntils 2022.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
-package com.wynntils.features;
+package com.wynntils.features.user;
 
 import static com.wynntils.mc.utils.InventoryUtils.MouseClickType.LEFT_CLICK;
 
-import com.wynntils.core.features.FeatureBase;
+import com.wynntils.core.features.UserFeature;
 import com.wynntils.core.features.properties.FeatureInfo;
-import com.wynntils.core.features.properties.GameplayImpact;
-import com.wynntils.core.features.properties.PerformanceImpact;
-import com.wynntils.core.features.properties.Stability;
+import com.wynntils.core.features.properties.FeatureInfo.GameplayImpact;
+import com.wynntils.core.features.properties.FeatureInfo.PerformanceImpact;
+import com.wynntils.core.features.properties.FeatureInfo.Stability;
+import com.wynntils.core.features.properties.KeyBinds;
 import com.wynntils.core.keybinds.KeyHolder;
 import com.wynntils.mc.utils.InventoryUtils;
 import com.wynntils.mc.utils.McUtils;
 import com.wynntils.wc.utils.WynnUtils;
 import org.lwjgl.glfw.GLFW;
 
+@KeyBinds
 @FeatureInfo(stability = Stability.STABLE, gameplay = GameplayImpact.MEDIUM, performance = PerformanceImpact.SMALL)
-public class IngredientPouchHotkeyFeature extends FeatureBase {
+public class IngredientPouchHotkeyFeature extends UserFeature {
+    public static IngredientPouchHotkeyFeature INSTANCE;
 
     private final KeyHolder ingredientPouchKeybind = new KeyHolder(
```

Comment:
It just struck me: can't we add annotations to the KeyHolder variables themselves instead? That'd be much nicer than using reflection to search for variables, which is kind of "magic". Something like `@RegisterKey`?

Reply:
Though about this when I first reviewed this too. I did not mention it because I feel like having every KeyHolder annotated separately could feel unnecessary. Why would a KeyHolder be declared in a feature that should not be registered? Then on the other hand, explicitly registering stuff works much nicer most of the times, so I am not against this.

Reply:
Yeah that sounds cool

Reply:
I like this idea, since it also parallels `@SubscribeEvent` somewhat